workers/data_refinery_workers/processors/gene_convert.R

```diff
@@ -0,0 +1,92 @@
+###
+# gene_convert.R
+# Alex's Lemonad Stand Foundation
+# Childhood Cancer Data Lab
+#
+###
+
+# import magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+#######################
+# The command interface!
+#######################
+
+library("optparse")
+library(data.table)
+library("dplyr")
+library("rlang")
+
+option_list = list(
+  make_option(c("-p", "--platform"), type="character", default="", 
+              help="Platform", metavar="character"),
+  make_option(c("-i", "--inputFile"), type="character", default="", 
+              help="inputFile", metavar="character"),
+  make_option(c("-o", "--outputFile"), type="character", default="", 
+              help="outputFile", metavar="character")
+); 
+
+opt_parser = OptionParser(option_list=option_list);
+opt = parse_args(opt_parser);
+
+platform <- opt$platform
+filePath <- opt$inputFile
+outFilePath <- opt$outputFile
+
+# Read the data file
+message("Reading data file...")
+suppressWarnings(data <- fread(filePath, 
+					stringsAsFactors=FALSE, 
+					sep="\t", header=TRUE, 
+					autostart=10, 
+					data.table=FALSE, 
+					check.names=FALSE, 
+					fill=TRUE, 
+					na.strings="", 
+					showProgress=FALSE)
+				)
+
+# Read the data file
+message("Reading master index...")
+index_path = paste0('zcat /home/user/gene_indexes/', platform, '.tsv.gz')
+suppressWarnings(index_data <- fread(index_path, 
+					stringsAsFactors=FALSE, 
+					sep="\t", 
+					header=TRUE, 
+					autostart=10, 
+					data.table=FALSE, 
+					check.names=FALSE, 
+					fill=TRUE, 
+					na.strings="", 
+					showProgress=FALSE)
+				)
+
+# find % overlap by column
+overlap <- apply(index_data, 2, function(x) length(intersect(data$ID_REF, x)) / length(data$ID_REF))
```

Comment:
This is also related to the header question -- if you assume that the first column will contain identifiers, could do `data[[1]]` instead of `data$ID_REF`